src/main/java/mate/academy/spring/config/DataInitializer.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.spring.config;
+
+import java.util.Set;
+import javax.annotation.PostConstruct;
+import mate.academy.spring.model.Role;
+import mate.academy.spring.model.User;
+import mate.academy.spring.service.RoleService;
+import mate.academy.spring.service.UserService;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DataInitializer {
+    private final RoleService roleService;
+    private final UserService userService;
+
+    public DataInitializer(RoleService roleService, UserService userService) {
+        this.roleService = roleService;
+        this.userService = userService;
+    }
+
+    @PostConstruct
+    public void inject() {
+        Role adminRole = new Role();
+        adminRole.setRoleName(Role.RoleName.ADMIN);
+        roleService.add(adminRole);
+        Role userRole = new Role();
+        userRole.setRoleName(Role.RoleName.USER);
+        roleService.add(userRole);
+        User user = new User();
+        user.setEmail("admin@i.ua");
+        user.setPassword("admin123");
+        user.setRoles(Set.of(adminRole));
```

Comment:
what about the `USER` role? I think we should add it too, at least for the testing purposes